src/runtime/test/description-test.ts

```diff
@@ -19,6 +19,7 @@ import {Recipe} from '../recipe/recipe.js';
 import {Relevance} from '../relevance.js';
 import {CollectionStorageProvider, VariableStorageProvider} from '../storage/storage-provider-base.js';
 import {FakeSlotComposer} from '../testing/fake-slot-composer.js';
+import { EntityType } from '../type.js';
```

Comment:
I've heard that we prefer to not use space here? Like so: {EntityType}

Reply:
done